web/themes/custom/recipe_theme/src/components/navbar.twig

```diff
@@ -0,0 +1,49 @@
+{#
+/**
+ * @file
+ * Template for Navbar component.
+ *
+ * Available config:
+ * - container: fixed | fluid.
+ * - placement: default | fixed-top | fixed-bottom | sticky-top
+ * - color: light | dark
+ * - utility_classes: An array of utility classes.
+ *
+ * Available blocks:
+ * - branding
+ * - left
+ * - right
+ */
+#}
+{{ attach_library('radix/navbar') }}
+
+{% set container = container == 'fixed' ? 'container' : false %}
+{% set placement = placement ?? '' %}
+
+<nav class="navbar navbar-expand-lg justify-content-between navbar-dark {{ placement }} {{ utility_classes|join(' ') }}">
```

Comment:
You hardcoded the dark color here, but you could have actually passed the color value at https://github.com/Pronovix/food/pull/52/files#diff-b2f684ce1ef5fd62d13500f925ecb33bR8 like this:

```
{% embed '@radix/navbar/navbar.twig' with {
    placement: 'sticky-top',
    container: 'fixed',
    color: 'dark',
    utility_classes: [],
  } %}
```

In that case I think you don't even need this file.